schemas/ephys/Ephys Rig Schema.json

```diff
@@ -0,0 +1,310 @@
+{
+	"$schema": "https://json-schema.org/draft/2020-12/schema",
+	"type": "object",
+	"properties": {
+		"version": {
+			"type": "string",
+			"const": "0.1.0",
+			"description": "schema version",
+			"title": "Version"
+		},
+		"rig_id": {
+			"type": "string",
+			"const": "0_0_0",
+			"description": "room_stim apparatus_version",
+			"title": "Rig ID"
+		},
+		"devices": {
+			"type": "object",
+			"properties": {
+				"probes": {
+					"type": "array",
+					"items": { "$ref": "#/$defs/probe" },
+					"uniqueItems": true,
+					"title": "Probes"
+				},
+				"cameras": {
+					"type": "array",
+					"items": {"$ref": "#/$defs/camera" },
+					"uniqueItems": true,
+					"title": "Cameras"
+				},
+				"lasers": {
+					"type": "array",
+					"items": {"$ref": "#/$defs/laser" },
+					"uniqueItems": true,
+					"title": "Lasers"
+				},
+				"visual_monitor": {
+					"type": "array",
+					"items": {"$ref": "#/$defs/monitor"},
+					"uniqueItems": true,
+					"title": "Visual monitor"
+				},
+				"running_disk": {
```

Comment:
Running disk is one where I can imagine there is actually no reason for there to be multiple ever.  Should we switch this one to not be an array?

Reply:
I can still be a reference to a subschema, just switch `type` to the `$ref` instead of `array` (I think).

Reply:
I think I found the syntax, but double check